compiler/rustc_borrowck/src/lib.rs

```diff
@@ -994,6 +997,23 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, '_, 'tcx> {
         }
     }
 
+    fn maybe_polonius_borrows_in_scope<'s>(
```

Comment:
why does this mention polonius at all? why isn't this just called borrows_in_scope?

Reply:
Yes, let's.

I am tucking the comment into this function, too.